lib/internal/buffer_old.js

```diff
@@ -11,6 +11,7 @@ const kMaxLength = smalloc.kMaxLength;
 exports.Buffer = Buffer;
 exports.SlowBuffer = SlowBuffer;
 exports.INSPECT_MAX_BYTES = 50;
+exports.kMaxLength = binding.kMaxLength;
```

Comment:
It's exposed because other libs like zlib need that value to know whether to error or not. Without it those libs wouldn't work properly in old buffer mode.